net-vpn/i2pd/i2pd-2.53.0.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake toolchain-funcs systemd
+
+DESCRIPTION="A C++ daemon for accessing the I2P anonymous network"
+HOMEPAGE="https://github.com/PurpleI2P/i2pd"
+SRC_URI="https://github.com/PurpleI2P/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="cpu_flags_x86_aes i2p-hardening +upnp"
+
+RDEPEND="
+	acct-user/i2pd
+	acct-group/i2pd
```

Comment:
Completely unrelated to the version bump, I just happened to notice this while reviewing it.

These acct-* packages are only needed for RDEPEND, not DEPEND. The distinction matters if you use --buildpkgonly. What do you think about demoting it? e.g.

```diff
diff --git a/net-vpn/i2pd/i2pd-2.52.0.ebuild b/net-vpn/i2pd/i2pd-2.52.0.ebuild
index 14ed8c998b07..4b92870e76b5 100644
--- a/net-vpn/i2pd/i2pd-2.52.0.ebuild
+++ b/net-vpn/i2pd/i2pd-2.52.0.ebuild
@@ -14,14 +14,16 @@ SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
 IUSE="cpu_flags_x86_aes i2p-hardening +upnp"
 
-RDEPEND="
-	acct-user/i2pd
-	acct-group/i2pd
+DEPEND="
 	dev-libs/boost:=
 	dev-libs/openssl:0=[-bindist(-)]
 	sys-libs/zlib
 	upnp? ( net-libs/miniupnpc:= )"
-DEPEND="${RDEPEND}"
+RDEPEND="
+	acct-user/i2pd
+	acct-group/i2pd
+	${DEPEND}
+"
 
 CMAKE_USE_DIR="${WORKDIR}/${P}/build"
 
```

Reply:
I agree :-) done!